content/blog/announcing-crossguard-preview/index.md

```diff
@@ -0,0 +1,95 @@
+
+Pulumi CrossGuard has something to offer everyone in your organization.
+
+### For Organization Administrators
+
+You want your organization to be able to move quickly and deliver business value. To do so, your team needs to be able to self-service their infrastructure. Pulumi CrossGuard gives you the peace of mind that you need to allow your team to work independently. Codify business and security policies to ensure your team meets the requirements you set forth.
+
+### For Security Engineers
+
+There's a lot of complexity to security and often the rest of your organization does not have all the knowledge they need to deliver software securely. Express security policies that allow you to share your knowledge and prevent issues from ending up in production.
+
+### For Developers and Operators
+
+There are a ton of moving pieces in our ecosystems today. It is difficult to know everything about the cloud resources we use. Pulumi CrossGuard ensures you do not fall into the trap of insecure or unreasonable defaults.
+
+## CrossGuard in Action
+
+With an understanding of what CrossGuard enables, let's look at what that means in practice. The following is a policy pack definition that prohibits creating an AWS S3 bucket that is public readable. Our Policy Pack is not complicated. It uses a `validateTypedResource` helper function to filter for the resource type we care about and reports a violation if present.
+
+```typescript
+new PolicyPack("aws-typescript", {
+    policies: [{
+        name: "s3-no-public-read",
+        description: "Prohibits setting the publicRead or publicReadWrite permission on AWS S3 buckets.",
+        enforcementLevel: "mandatory",
+        validateResource: validateTypedResource(aws.s3.Bucket, (bucket, args, reportViolation) => {
+            if (bucket.acl === "public-read" || bucket.acl === "public-read-write") {
+                reportViolation(
+                    "You cannot set public-read or public-read-write on an S3 bucket. " +
+                    "Read more about ACLs here: https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html");
+            }
+        }),
+    }],
+});
+```
+
+We initially run `pulumi preview` for an S3 bucket that will pass the policy.
+
+```typescript
+// Create an AWS resource (S3 Bucket)
+const bucket = new aws.s3.Bucket("my-bucket");
+```
+
+We then modify the bucket to have its access control list (ACL) set to `public-read`, which will cause our bucket to be out-of-compliance.
+
+```typescript
+// Create an AWS resource (S3 Bucket)
+const bucket = new aws.s3.Bucket("my-bucket", {
+    acl: "public-read",
+});
+```
+
+![Demo of Policy as Code](pac-demo.gif)
+
+While this is a simple demonstration of what you can do with Policy as Code, the framework is very flexible and supports much more complex policies. Our [AWSGuard library](https://github.com/pulumi/pulumi-awsguard) is a great example of how you can use Policy as Code to write configurable policies.
```

Comment:
I realize this is late-breaking, however we would like to message that OPA is part of our thinking here. We will be opening up the OPA repo -- similar to AWSGuard -- as an _example_ of the sort of things enabled.

So given that, what do you think about this sentence:

> Our [OPAGuard library](https://github.com/pulumi/pulumi-opaguard) demonstrates an alternative capability where CrossGuard can leverage existing ecosystems of rules, in this case any Open Policy Agent (OPA) collection of rules.

Reply:
I dont have access to the OPAGuard library -- can you grant me access?